Source/Core/Core/Boot/Boot.cpp

```diff
@@ -255,9 +255,11 @@ bool CBoot::FindMapFile(std::string* existing_map_file, std::string* writable_ma
     *writable_map_file = File::GetUserPath(D_MAPS_IDX) + game_id + ".map";
 
   bool found = false;
-  static const std::string maps_directories[] = {File::GetUserPath(D_MAPS_IDX),
-                                                 File::GetSysDirectory() + MAPS_DIR DIR_SEP};
-  for (size_t i = 0; !found && i < ArraySize(maps_directories); ++i)
+  static const std::array<std::string, 2> maps_directories{
+      File::GetUserPath(D_MAPS_IDX),
+      File::GetSysDirectory() + MAPS_DIR DIR_SEP,
+  };
+  for (size_t i = 0; !found && i < maps_directories.size(); ++i)
```

Comment:
Couldn't this be a ranged for-loop with an immediate `return true` inside the loop?

Reply:
Amended the commit with that suggestion